schemas/common_definitions.json

```diff
@@ -302,6 +302,14 @@
     "type": "string",
     "minLength": 1
   },
+  "string_array": {
+    "type": "array",
+    "items": {
+      "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+    },
+    "uniqueItems": true,
+    "minLength": 1
```

Comment:
Min length is no longer needed now that you validate the answer ids but `minItems` is needed as currently an empty array would validate successfully. 
https://json-schema.org/understanding-json-schema/reference/array.html#length